api/src/main/java/org/pmiops/workbench/access/AccessTierService.java

```diff
@@ -63,8 +63,6 @@ public interface AccessTierService {
    */
   List<String> getAccessTierShortNamesForUser(DbUser user);
 
-  List<DbUser> getAllRegisteredTierUsers();
```

Comment:
no longer used by anything.  if we need it again, it's probably better to make the new version tier-generic